### ISSUE TITLE
plugin w/ custom rule: what am I doing wrong?

### ISSUE THREAD

**Issue by boneskull** [eslint/eslint#4627]
Given [this package](/boneskull/eslint-plugin-prototype-chain), running ESLint after installing it results in this:

```
 1:1  warning  Definition for rule 'prototype-chain' was not found  prototype-chain
```

Here's my `.eslintrc`:

```yaml
---
plugins:
  - prototype-chain

parser: babel-eslint

rules:
  space-before-function-paren: 0
  arrow-parens:
    - 1
    - as-needed
  prototype-chain: 1
```

**Comment by eslintbot** [comment]
@boneskull Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by boneskull** [comment]
```shell
$ eslint -v
v1.10.3
```

**Comment by btmills** [comment]
Check out the configuration section for [eslint-plugin-react](https://github.com/yannickcr/eslint-plugin-react#configuration) as an example. To configure plugin rules, first specify the plugin name, then the rule, i.e. `"plugin-name/plugin-rule": 1`. In your case, it looks like the plugin and the rule have the same name, so the rule configuration would be `"prototype-chain/prototype-chain": 1`. Does that help?
